interface/main/holidays/import_holidays.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/**
+ * interface/main/holidays/import_holidays.php holidays/clinic handle import/download holidays files
+ *
+ * Copyright (C) 2016 Sharon Cohen <sharonco@matrix.co.il>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  sharonco <sharonco@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+//this part download the CSV file after the click on the href link
+if($_GET['download_file']==1)
+{
+    if ( ! file_exists($_GET['filename']))
```

Comment:
Also, shouldn't this go below the authentication code (after the require_once of interface/globals.php), otherwise this part of this script requires no authentication.